### ISSUE TITLE
Version 1.3.0 breaks .ipa deployment

### ISSUE THREAD

**Issue by akashivskyy** [bitrise-io/steps-deploy-to-bitrise-io#32]
After updating to 1.3.0, the step started to fail when deploying an `.ipa` app:

```none
analyzing ipa
Deploy failed, error: failed to unwrap Info.plist from ipa, error: failed to find Info.plist
```

It worked perfectly well in version 1.2.9. Is this a regression or perhaps a misconfiguration on my side?

**Comment by godrei** [comment]
Hi @akashivskyy ,
thanks for the report, i'm working on the issue.
Until i release the fix, please use the previous version (1.2.9).

**Comment by akashivskyy** [comment]
Ok! Waiting for updates. :)

**Comment by godrei** [comment]
@akashivskyy we released a new version ([1.3.1](https://github.com/bitrise-io/steps-deploy-to-bitrise-io/releases/tag/1.3.1)) with the fix. Thanks for the report!

I'll close this issue, but if you need more info/help just let us know and we'll reopen it!
